src/quantum/plugin/modifications/StatementModifaction.ts

```diff
@@ -41,8 +41,9 @@ export class StatementModification {
                     statement.setFunctionName('$fsx.r');
                     statement.setValue(resolvedFile.getID());
                 } else {
-
-                    if (core.opts.isTargetServer() || core.opts.isTargetUniveral()) {
+                    if(core.opts.isTargetNpm()) {
+                        statement.setFunctionName('require');
```

Comment:
Please test it, because there is no need in settings to `require`, as we are transforming from `require`. Long story short, no changes to the statement function